src/app/add-modules/add-module-modal/add-module-modal.component.html

```diff
@@ -1,12 +1,12 @@
-<div class="add-module-dialog">
+<div class="add-dialog">
   <p mat-dialog-title class="text-light-black">Add Module</p>
   <p class="text-hint-color">Lorem ipsum dolor sit amet.</p>
-  <div fxLayout="column" fxLayoutAlign="flex-start" mat-dialog-actions class="module-entry">
-    <p class="module-mane text-hint-color">Module Name</p>
-    <input class="module-input p-16 pr-312 text-light-black bg-light-color" type="text" placeholder="Lorem ipsum dolor sit amet.">
-    <div fxLayout="row" fxLayoutAlign="flex-end" class="dialog-btn mt-24">
-      <button mat-button mat-dialog-close class="close-btn text-primary-color">CLOSE</button>
-      <button mat-button mat-dialog-close class="add-btn bg-primary-color text-white-color">ADD</button>
+  <div fxLayout="column" fxLayoutAlign="flex-start" mat-dialog-actions class="add-dialog__content">
+    <p class="text-hint-color">Module Name</p>
+    <input class="add-dialog__content-input p-16 text-light-black bg-light-color" ngClass.xs="pr-100" ngClass.gt-xs="pr-312" type="text" placeholder="Lorem ipsum dolor sit amet.">
+    <div fxLayout="row" fxLayoutAlign="flex-end" class="mt-24">
+      <button mat-button mat-dialog-close class="add-dialog__close-btn text-primary-color hand-cursor">CLOSE</button>
+      <button mat-button mat-dialog-close class="add-dialog__add-btn bg-primary-color text-white-color hand-cursor">ADD</button>
```

Comment:
use`mat-raised-button`
```suggestion
      <button mat-raised-button color="primary" class="add-dialog__add-btn>ADD</button>
     
```